src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,197 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTORY = 0.75f;
+    private static final int MAX_CAPACITY = 1 << 30;
+
+    private Node<K, V>[] mapTable;
+    private int size;
+    private final float loadFactor;
+    private int threshold;
+
+    public MyHashMap() {
+        this.loadFactor = DEFAULT_LOAD_FACTORY;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        putValue(hashCode(key), key, value, false);
     }
 
     @Override
     public V getValue(K key) {
-        return null;
+        Node<K, V> node;
+        return (node = getNode(key)) == null ? null : node.value;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    static int hashCode(Object key) {
```

Comment:
```suggestion
    private int hashCode(Object key) {
```
![image](https://user-images.githubusercontent.com/20326858/224001407-cf3c16fd-3819-4334-9bda-6eb02e25ac27.png)